.github/workflows/kantra.yaml

```
@@ -0,0 +1,123 @@
+        uses: actions/setup-go@v6
+        with:
+          go-version: '1.25'
+
+      - name: Build Koncur
+        shell: bash
+        run: |
+          go build -o $HOME/.local/bin/koncur ./cmd/koncur/main.go
+
+      - name: Get Token
+        id: get_workflow_token
+        if: ${{ !fromJSON(inputs.skip_maven) }}
```

actionlint (1.7.10)

1st argument of function call is not assignable. "bool" cannot be assigned to "string". called function type is "fromJSON(string) -> any"

(expression)